server/store/sqlstore/public_methods.go

```diff
@@ -0,0 +1,342 @@
+	return s.getPostInfoByMSTeamsID(s.db, chatID, postID)
+
+}
+
+func (s *SQLStore) GetPostInfoByMattermostID(postID string) (*storemodels.PostInfo, error) {
+	return s.getPostInfoByMattermostID(s.db, postID)
+
+}
+
+func (s *SQLStore) GetStats(remoteID string, preferenceCategory string) (*storemodels.Stats, error) {
+	return s.getStats(s.db, remoteID, preferenceCategory)
+
+}
+
+func (s *SQLStore) GetSubscriptionType(subscriptionID string) (string, error) {
+	return s.getSubscriptionType(s.db, subscriptionID)
+
+}
+
+func (s *SQLStore) GetSubscriptionsLastActivityAt() (map[string]time.Time, error) {
+	return s.getSubscriptionsLastActivityAt(s.db)
+
+}
+
+func (s *SQLStore) GetTokenForMSTeamsUser(userID string) (*oauth2.Token, error) {
+	return s.getTokenForMSTeamsUser(s.db, userID)
+
+}
+
+func (s *SQLStore) GetTokenForMattermostUser(userID string) (*oauth2.Token, error) {
+	return s.getTokenForMattermostUser(s.db, userID)
+
+}
+
+func (s *SQLStore) GetUserConnectStatus(mmUserID string) (*storemodels.UserConnectStatus, error) {
+	return s.getUserConnectStatus(s.db, mmUserID)
+
+}
+
+func (s *SQLStore) GetWhitelistCount() (int, error) {
+	return s.getWhitelistCount(s.db)
+
+}
+
+func (s *SQLStore) GetWhitelistEmails(page int, perPage int) ([]string, error) {
+	return s.getWhitelistEmails(s.db, page, perPage)
+
+}
+
+func (s *SQLStore) IsUserWhitelisted(userID string) (bool, error) {
+	return s.isUserWhitelisted(s.db, userID)
+
+}
+
+func (s *SQLStore) LinkPosts(postInfo storemodels.PostInfo) error {
+	return s.linkPosts(s.db, postInfo)
+
+}
+
+func (s *SQLStore) ListChannelLinks() ([]storemodels.ChannelLink, error) {
+	return s.listChannelLinks(s.db)
+
+}
+
+func (s *SQLStore) ListChannelLinksWithNames() ([]*storemodels.ChannelLink, error) {
+	return s.listChannelLinksWithNames(s.db)
+
+}
+
+func (s *SQLStore) ListChannelSubscriptions() ([]*storemodels.ChannelSubscription, error) {
+	return s.listChannelSubscriptions(s.db)
+
+}
+
+func (s *SQLStore) ListChannelSubscriptionsToRefresh(certificate string) ([]*storemodels.ChannelSubscription, error) {
+	return s.listChannelSubscriptionsToRefresh(s.db, certificate)
+
+}
+
+func (s *SQLStore) ListChatSubscriptionsToCheck() ([]storemodels.ChatSubscription, error) {
+	return s.listChatSubscriptionsToCheck(s.db)
+
+}
+
+func (s *SQLStore) ListGlobalSubscriptions() ([]*storemodels.GlobalSubscription, error) {
+	return s.listGlobalSubscriptions(s.db)
+
+}
+
+func (s *SQLStore) ListGlobalSubscriptionsToRefresh(certificate string) ([]*storemodels.GlobalSubscription, error) {
+	return s.listGlobalSubscriptionsToRefresh(s.db, certificate)
+
+}
+
+func (s *SQLStore) MattermostToTeamsUserID(userID string) (string, error) {
+	return s.mattermostToTeamsUserID(s.db, userID)
+
+}
+
+func (s *SQLStore) RecoverPost(postID string) error {
+	return s.recoverPost(s.db, postID)
+
+}
+
+func (s *SQLStore) SaveChannelSubscription(subscription storemodels.ChannelSubscription) error {
```

Comment:
What makes these "different"?

Reply:
Those are the ones that run multiple DB queries, so they get their transactions managed

Reply:
Ah, yes, somehow I misread this -- that's obvious from context now, thank you!